src/docfx/config/ops/OpsMetadataRuleConverter.cs

```diff
@@ -117,6 +123,7 @@ private static bool TryGetAttributeAdditionalErrors(Dictionary<string, OpsMetada
                 { "Date", new string[] { "date-format-invalid", "date-out-of-range" } },
                 { "MicrosoftAlias", new string[] { "ms-alias-invalid" } },
                 { "Deprecated", new string[] { "attribute-deprecated" } },
+                { "Uniqueness", new string[] { "attribute-duplicated-within-docset" } },
```

Comment:
Given the error code in feature spec is `duplicate-h1`, `duplicate-title`, how about calling it `duplicate-attribute`? 
```suggestion
                { "Uniqueness", new string[] { "duplicate-attribute" } },
```

Reply:
just follow the existing error code defined in docfx, like attribute-deprecated, I am OK with "duplicate-attribute"

Reply:
It is called `attribute-deprecated` because the actual error code is like `title-deprecated` or `msalias-deprecated`, see the pattern 🤣

Reply:
the pattern is no pattern here 🤣🤣🤣